package.json

```
@@ -3,12 +3,14 @@
   "version": "1.0.0",
   "main": "eslint.config.js",
   "packageManager": "npm@11.5.2",
+  "type": "commonjs",
   "workspaces": [
     "src/examples/standalone/vercel",
     "src/examples/telegram/vercel"
```

Because the workspace is not registered, the turbo command at the root level (`npm run start:email`) doesn't work
```suggestion
    "src/examples/telegram/vercel",
    "src/examples/email"
```